### ISSUE TITLE
Synctex will not work if address contains unicode characters.

### ISSUE THREAD

**Issue by jenickp** [texstudio-org/texstudio#400]
If there are unicode characters in address of the folder containing .tex file, synctex will not work in both direction (source->pdf & pdf->source).

### Environment
- TeXstudio: 2.12.10
- Qt: 5.8.0 R
- OS: Windows 10
- TeX distribution: texlive 2018

### How to reproduce
Put a unicode character (e.g. β) in folder address.

**Comment by sunderme** [comment]
works fine on linux ...

**Comment by sunderme** [comment]
works on win10 with miktex

**Comment by sunderme** [comment]
in the path ß, there should be a file document.synctex.gz. Please upload that file.

**Comment by jenickp** [comment]
[folder ß](https://1drv.ms/f/s!AtfPusRXx6Z-hg_YnGHApFfreESU)

**Comment by GitHubRulesOK** [comment]
For what its worth In this example the synctex file is valid and works as expected in another 2.12.10
![untitled2](https://user-images.githubusercontent.com/12754913/52642428-e74b6a80-2ed2-11e9-8ac6-da65ef6c98ca.png)

IT WILL also work in any other editor or viewer for forward and inverse search.
Thus it is not the synctex file at fault. It is either the editor not calling the viewer correctly via synctex or there is something stopping the viewer seeing the request to highlight line 12. NOTE the tex file must be in the directory as specified (spaces are NOT allowed, and **Unicode especially "composed characters" is to be discouraged**, if the ß is a single character (not made from 2) it may be acceptable as on my system, however its a grey area when accents are overstruck on Latin characters and **CJK is a nightmare waiting to fail**)
![untitled](https://user-images.githubusercontent.com/12754913/52641549-047f3980-2ed1-11e9-976e-f1d6aae3ba36.png)

**Comment by tausackhn** [comment]
Default TeXworks doesn't have such issue on the other hand. It works correctly with paths containing unicode characters.
### Environment
-  TeXstudio: 3.0.5
-  Qt: 5.15.2 R
-  OS: Windows 10 (En-Ru locale)
-  TeX distribution: MikTeX 2.9 (packages on 2021.02.17)

**Comment by vaggelisfelouzis-web** [comment]
I agree Inverse search fails silently on Windows when file names or paths contain Greek characters. Other editors like Kile or TeXworks handle this correctly on the same machine. Of course Renaming to Latin fixes the issue but you must know the solution....

**Comment by GitHubRulesOK** [comment]
@vaggelisfelouzis-web ensure you are using current PRE-release 3.7 as there were several changes to better filepath support including code page and UTF8
